src/Document/TaxonDocument.php

```diff
@@ -29,7 +31,7 @@ class TaxonDocument
      *
      * @ElasticSearch\Property(type="integer")
      */
-    private $position;
+    private $position = 0;
```

Comment:
If position is not set or is null, this is an integer column and Elastic can't have null values. So we set it by default.